core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -35,6 +35,11 @@
 @Table(name = "MESSAGE_DESTINATIONS")
 public class MessageDestination extends EntityWithUuid
 {
+	public enum TypeOfDestination
+	{
+		Named, Anonymous
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS00115)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS00115)